executor/index_lookup_hash_join.go

```diff
@@ -292,6 +292,11 @@ func (e *IndexNestedLoopHashJoin) Close() error {
 		}
 		e.taskCh = nil
 	}
+	if e.runtimeStats != nil {
+		rootStats := e.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl.GetRootStats(e.baseExecutor.id.String())
+		concurrency := len(e.joinChkResourceCh)
```

Comment:
`len` for a channel means  the number of elements queued (unread) in the channel buffer.
We should use `cap` here.

The same as below.